src/Services/SearchService.php

```diff
@@ -531,8 +532,8 @@ private function validateResponse($responseData)
         if (is_array($responseData) && array_key_exists('error', $responseData) && $responseData['error'] === true) {
             $errorMsg = 'Plentymarkets SDK returned an error response';
         } elseif (!is_string($responseData)) {
-            $errorMsg = 'Invalid response received from server';
-        } elseif ($responseData === '') {
+            $errorMsg = 'Plentymarkets SDK returned invalid response - Expected string';
```

Comment:
Maybe to make this more clear, by logging what the type actually was.
```suggestion
            $errorMsg = sprintf('Plentymarkets SDK returned invalid response (expected string, received %s)', gettype($responseData));
```

Reply:
Tried `gettype` before and guess what, Plenty doesn't allow it...

Reply:
Great. Nvm then